drivers/python/rethinkdb/net.py

```diff
@@ -515,6 +523,12 @@ def __init__(self, conn_type, host, port, db, auth_key, timeout, ssl, **kwargs):
         self._instance = None
         self._next_token = 0
 
+    def client_port(self):
+        return self._instance._socket.client_port()
```

Comment:
Eh... we don't use properties anywhere currently in the js driver, and I don't think this is important enough to warrant it. Generally I think properties are useful if you need to be backwards compatible with something that was previously an attribute. I think it being a function works fine in js. (Incidentally, I think making it a read-only property is overkill in python too, given python's "we're all adults here" ethos)